scripts/release/changelog.js

```diff
@@ -32,7 +32,7 @@ const optionsForBuilds = Object.assign( {}, commonOptions, {
 
 Promise.resolve()
 	.then( () => devEnv.generateChangelogForSubRepositories( optionsForDependencies ) )
-	.then( () => devEnv.generateSummaryChangelog( optionsForBuilds ) )
+	.then( response => devEnv.generateSummaryChangelog( Object.assign( optionsForBuilds, response ) ) )
```

Comment:
Isn't `Object.assign( {}, optionsForBuilds, response )` safer? This modifies the `optionsForBuilds` object.

Reply:
Which is defined a few lines above. It's saffe enough in this case.